notebooks/data-sources/TestGrid/testgrid_metadata_EDA.ipynb

```diff
@@ -283,7 +283,7 @@
      "name": "stdout",
```

Comment:
<p>Why did you decide to use a non-red hat set of tests here? And what is jetstack?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Looking at other testgrid dashboards eg like jetstack, helps us identify the purpose of metadata fields like "custom-columns" which are empty or non populated in the openshift dashboards. Looking at another dashboard has no real significance here other than trying to understand the significance of certain metadata columns which are normally not populated in the openshift dashboards. Since the goal was to understand the various Testgrid metadata fields, we decided to choose dissimilar dashboards hoping that they'd give us wider insight into the kind of data captured by those fields.</p><p> </p>